src/main/java/eu/siacs/conversations/ui/ConversationsActivity.java

```diff
@@ -515,11 +515,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.action_scan_qr_code:
                 UriHandlerActivity.scan(this);
                 return true;
+            case R.id.select_all_main:
+                selectAllConversations(true);
+                return true;
             case R.id.delete_button:
                 wipeSelectedConversations();
                 return true;
             case R.id.select_all:
-                selectAllConversations();
+                selectAllConversations(false);
```

Comment:
I got following error when I click this button

`java.lang.NullPointerException: Attempt to read from field 'eu.siacs.conversations.databinding.ConversationListRowBinding eu.siacs.conversations.ui.adapter.ConversationAdapter$ConversationViewHolder.binding' on a null object reference
        at eu.siacs.conversations.ui.ConversationsOverviewFragment.changeBackgroundColor(ConversationsOverviewFragment.java:394)
        at eu.siacs.conversations.ui.ConversationsActivity.selectAllConversations(ConversationsActivity.java:563)
        at eu.siacs.conversations.ui.ConversationsActivity.onOptionsItemSelected(ConversationsActivity.java:519)
`